src/ticks/debugger.c

```diff
@@ -3,6 +3,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <ctype.h>
+#include <unistd.h>                         // For declarations of isatty()
```

Comment:
For WIN32 we can `#include <io.h>` which defines `isatty()` rather than `unistd.h`. This should work on both mingw and MSVC